### DISCUSSION TITLE
Will HyperHDR installed on a Windows PC send signals to an Adafruit Feather RP2040 Scorpio via USB?

### DISCUSSION THREAD

**Discussion by markgabrang** [awawa-dev/HyperHDR#906] [Q&A] [answered]
Hi again @awawa-dev !

Basically the title says it all.
I'm looking at the possibility of skipping the RPI entirely, and run HyperHDR on the PC:

PC HDMI > MS2130 > PC USB PORT for video capture
PC USB PORT > Adafruit Feather RP2040 Scorpio > HD107S for data

Is PC build without a Raspberry Pi 4/5 possible?

Thank you so much.

**Comment by awawa-dev** [comment, marked as answer]
Hi
Yes, it will work

**Reply by markgabrang** [reply]
Thank you so much!

I could also just use the internal DX11 capture to save on parts, right?

**Reply by awawa-dev** [reply]
Yes, only DRM apps like Netflix wont allow capturing. We live in a time when corporations want us to have nothing, even if we paid for it.

**Reply by markgabrang** [reply]
That's wild. 
It's almost always better to get the MKV versions anyway. 😏

Again thank you so much for the work that you do for everyone! 🙏

**Reply by markgabrang** [reply]
Hi again @awawa-dev , does unchecking "enable hardware acceleration" means HyperHDR will offload capture to CPU?

**Reply by awawa-dev** [reply]
Yes, CPU will need to handle image scaling and colorspace conversion.

**Reply by markgabrang** [reply]
So no GPU overhead with HW acceleration off then? Cool.

Forgive my enthusiasm, I can't sit still while the parts are en route.

**Reply by awawa-dev** [reply]
I recommend that you do not disable this option and leave these tasks to the GPU. Without this, even if the GPU has less workload, much more data will be transferred over PCI-E because GPU scaling will not be active. Also automatic HDR to SDR conversion wont work.

**Reply by markgabrang** [reply]
Got it. In my case, the GPU has a lot of headroom so it doesn't matter either way.

Appreciate it!
